packages/moonstone/Scrollable/ScrollButtons.js

```diff
@@ -236,36 +236,50 @@ class ScrollButtons extends Component {
 	onSpotlight = (ev) => {
 		const
 			{focusableScrollButtons, rtl, vertical} = this.props,
-			{target} = ev;
+			{target} = ev,
+			direction = getDirection(ev.keyCode),
+			isDown = direction === 'down',
+			isLeftMovement = direction === (rtl ? 'right' : 'left'),
+			isRightMovement = direction === (rtl ? 'left' : 'right'),
+			isUp = direction === 'up',
+			isNextButton = target === this.nextButtonRef.current,
+			isPrevButton = target === this.prevButtonRef.current;
```

Comment:
Before executing the following code, it seems to me that we need to guard the code with `direction` condition.
```
if (direction) {
...
}
```

Reply:
I thought it is safe for now since _onSpotlight[Down|Left|Right|Up]_ func props are called only for arrow keys. I will add guards here.

Reply:
👍